src/app/page.tsx

```diff
@@ -289,7 +313,37 @@ export default function Home() {
           {v_correctionPaths && (
               <div>
                 {v_correctionPaths}
-                <button onClick={() => handleVDelete()}>Delete Vignetting Correction File</button>
+                <button onClick={() => handle_v_delete()}>Delete Vignetting Correction File</button>
+               </div>
+          )}
+        </div>
+        <h2>Neutral Density Correction Path Upload</h2>
+        <button
+          onClick={dialogND}
+          className="bg-gray-300 hover:bg-gray-400 text-gray-700 font-semibold py-1 px-2 border-gray-400 rounded"
+        >
+          Select Neutral Density Correction Files
```

Comment:
One other thought... would `File` and not `Files` be a better fit for this button? Just a small thing, it doesn't really matter at this stage either way :)